analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/Descriptors/RuleParameter.cs

```diff
@@ -19,31 +19,49 @@
  */
 
 using System.Diagnostics.CodeAnalysis;
+#if NETCOREAPP  // ToDo: Remove conditional compilation after change of target framework
+using System.Text.Json.Serialization;
+#endif
 using System.Xml;
 using System.Xml.Serialization;
+using SonarAnalyzer.Common;
 
-namespace SonarAnalyzer.RuleDescriptorGenerator
+namespace SonarAnalyzer.RuleDescriptorGenerator;
+
+[ExcludeFromCodeCoverage]
+public class RuleParameter
 {
-    [ExcludeFromCodeCoverage]
-    public class RuleParameter
+    // ToDo: Remove XML annotations
+    [XmlElement("key")]
+    public string Key { get; set; }
+
+    [XmlIgnore]
```

Comment:
As the description is ignored for the xml, shouldn`t it be also ignored for the json?